volmdlr/primitives3d.py

```diff
@@ -401,6 +401,17 @@ def __init__(self, plane_origin: volmdlr.Point3D,
         volmdlr.faces.ClosedShell3D.__init__(self, faces, color=color,
                                              alpha=alpha, name=name)
 
+    def copy(self, deep=True, memo=None):
+        return self.__class__(plane_origin=self.plane_origin.copy(),
+                              x=self.x.copy(),
+                              y=self.y.copy(),
+                              outer_contour2d=self.outer_contour2d.copy(),
+                              inner_contours2d=[c.copy() for c in self.inner_contours2d],
+                              extrusion_vector=self.extrusion_vector.copy(),
+                              color=self.color,
+                              alpha=self.alpha,
+                              name=self.name)
```

Comment:
It might add some bug if all the copy methods don't have the deep and memo attributes. 
```suggestion
    def copy(self, deep=True, memo=None):
        return self.__class__(plane_origin=self.plane_origin.copy(deep=deep, memo=memo),
                              x=self.x.copy(deep=deep, memo=memo),
                              y=self.y.copy(deep=deep, memo=memo),
                              outer_contour2d=self.outer_contour2d.copy(deep=deep, memo=memo),
                              inner_contours2d=[c.copy(deep=deep, memo=memo) for c in self.inner_contours2d],
                              extrusion_vector=self.extrusion_vector.copy(deep=deep, memo=memo),
                              color=self.color,
                              alpha=self.alpha,
                              name=self.name)
```